IPython/external/mathjax.py

```diff
@@ -1,237 +1,13 @@
-
-    Parameters
-    ----------
-
-    replace : bool [False]
-        Whether to remove and replace an existing install.
-    dest : str [IPYTHONDIR/nbextensions/mathjax]
-        Where to install mathjax
-    tag : str ['2.4.0']
-        Which tag to download. Default is '2.4.0', the current stable release,
-        but alternatives include 'v1.1a' and 'master'.
-    file : file like object [ defualt to content of https://github.com/mathjax/MathJax/tarball/#{tag}]
-        File handle from which to untar/unzip/... mathjax
-    extractor : function
-        Method to use to untar/unzip/... `file`
-    """
-    try:
-        anything_to_do = prepare_dest(dest, replace)
-    except OSError as e:
-        print("ERROR %s, require write access to %s" % (e, dest))
-        return 1
-    else:
-        if not anything_to_do:
-            return 0
-
-    if file is None:
-        # download mathjax
-        mathjax_url = "https://github.com/mathjax/MathJax/archive/%s.tar.gz" %tag
-        print("Downloading mathjax source from %s" % mathjax_url)
-        response = urlopen(mathjax_url)
-        file = response.fp
-
-    print("Extracting to %s" % dest)
-    extractor(file, dest)
-    return 0
-
-
-def main():
-    parser = argparse.ArgumentParser(
-            description="""Install mathjax from internet or local archive""",
-    )
-
-    parser.add_argument(
-            '-i',
-            '--install-dir',
-            default=nbextensions,
-            help='custom installation directory. Mathjax will be installed in here/mathjax')
-
-    parser.add_argument(
-            '-d',
-            '--print-dest',
-            action='store_true',
-            help='print where mathjax would be installed and exit')
-    parser.add_argument(
-            '-r',
-            '--replace',
-            action='store_true',
-            help='Whether to replace current mathjax if it already exists')
-    parser.add_argument('filename',
-            help="the local tar/zip-ball filename containing mathjax",
-            nargs='?',
-            metavar='filename')
-
-    pargs = parser.parse_args()
-
-    dest = os.path.join(pargs.install_dir, 'mathjax')
-
-    if pargs.print_dest:
-        print(dest)
-        return
-
-    # remove/replace existing mathjax?
-    replace = pargs.replace
-
-    # do it
-    if pargs.filename:
-        fname = pargs.filename
-
-        # automatically detect zip/tar - could do something based
-        # on file content, but really not cost-effective here.
-        if fname.endswith('.zip'):
-            extractor = extract_zip
-        else :
-            extractor = extract_tar
-        # do it
-        return install_mathjax(file=open(fname, "rb"), replace=replace, extractor=extractor, dest=dest)
-    else:
-        return install_mathjax(replace=replace, dest=dest)
+"""
 
+import sys 
 
 if __name__ == '__main__' :
-    sys.exit(main())
+    sys.exit("IPython.external.mathjax is deprecated")
```

Comment:
Do you want to explain that mathjax is already installed, not just that "what you want is gone"?